src/main/java/org/jabref/model/entry/identifier/DOI.java

```diff
@@ -43,20 +44,46 @@
             + "[/:]"                            // divider
             + "(?:[^\\s]+)"                     // suffix alphanumeric without space
             + ")";                              // end group \1
+
+    // Regex (Short DOI)
+    private static final String SHORT_DOI_EXP = ""
+            + "(?:urn:)?"                       // optional urn
+            + "(?:doi:)?"                       // optional doi
+            + "("                               // begin group \1
+            + "10"                              // directory indicator
+            + "[/:]"                            // divider
+            + "[a-z0-9]+"
+            + ")";                              // end group \1
+    private static final String FIND_SHORT_DOI_EXP = ""
+            + "(?:urn:)?"                       // optional urn
+            + "(?:doi:)?"                       // optional doi
+            + "("                               // begin group \1
+            + "10"                              // directory indicator
+            + "[/:]"                            // divider
+            + "[a-z0-9]+"
+            + "(?:[^\\s]+)"                     // suffix alphanumeric without space
+            + ")";                              // end group \1
+
     private static final String HTTP_EXP = "https?://[^\\s]+?" + DOI_EXP;
+    private static final String SHORT_DOI_HTTP_EXP = "https?://[^\\s]+?" + SHORT_DOI_EXP;
     // Pattern
     private static final Pattern EXACT_DOI_PATT = Pattern.compile("^(?:https?://[^\\s]+?)?" + DOI_EXP + "$", Pattern.CASE_INSENSITIVE);
     private static final Pattern DOI_PATT = Pattern.compile("(?:https?://[^\\s]+?)?" + FIND_DOI_EXP, Pattern.CASE_INSENSITIVE);
+    // Pattern (short DOI)
+    private static final Pattern EXACT_SHORT_DOI_PATT = Pattern.compile("^(?:https?://[^\\s]+?)?" + SHORT_DOI_EXP, Pattern.CASE_INSENSITIVE);
+    private static final Pattern SHORT_DOI_PATT = Pattern.compile("(?:https?://[^\\s]+?)?" + FIND_SHORT_DOI_EXP, Pattern.CASE_INSENSITIVE);
     // DOI
     private final String doi;
+    // Short DOI
+    private boolean shortDoi;
```

Comment:
rename the variable to `isShortDoi` please

Reply:
Sure. Renamed.